python_ta/reporters/html_reporter.py

```diff
@@ -3,6 +3,7 @@
 import sys
 
 from jinja2 import Environment, FileSystemLoader
+from markdown_it import MarkdownIt
```

Comment:
This library needs to be added to the project requirements (you'll need to find where these are specified)